mappings/1.11.2#17w06a.tinydiff

```diff
@@ -39,7 +39,7 @@ c	net/minecraft/unmapped/C_5904422
 	m	(Lnet/minecraft/unmapped/C_9590849;)Z	m_9015927		m_9015927
 		p	1			p_1
 c	net/minecraft/unmapped/C_6260518
-	m	(Lnet/minecraft/unmapped/C_8105098;I)V	m_4993396		m_4993396
+	m	(Lnet/minecraft/unmapped/C_8105098;I)V	m_4993396		saveOrLoadToolbar
```

Comment:
I'm not able to look at the code right now so this name is a little confusing to me. Is it like a toggle?

Reply:
code is basically ```java
boolean save = saveKey.isPressed();
boolean load = loadKey.isPressed();
saveOrLoadToolBar(..., save, load)```
(at least in 1.12.2)

Reply:
I see, so it's a bit of a mojank moment then.